apparmor.d/groups/network/dhcpcd

```diff
@@ -0,0 +1,64 @@
+# Last Modified: Sun Feb 19 16:26:38 2023
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{usr/,}bin/dhcpcd
+profile dhcpcd @{exec_path} flags=(attach_disconnected, mediate_deleted) {
+  include <abstractions/base>
+
+  @{exec_path} mr,
+  
+  capability dac_override,
+  capability dac_read_search,
+  capability kill,
+  capability net_admin,
+  capability net_bind_service,
+  capability net_raw,
+  capability setgid,
+  capability setuid,
+  capability sys_chroot,
+
+  network inet dgram,
+  network inet raw,
+  network inet6 dgram,
+  network inet6 raw,
+  network netlink raw,
+  network packet raw,
+
+  /{usr/,}bin/bash ix,
+  /{usr/,}bin/cat mrix,
+  /{usr/,}bin/chmod mrix,
+  /{usr/,}bin/cmp mrix,
+  /{usr/,}bin/mkdir mrix,
+  /{usr/,}bin/rm mrix,
+  /{usr/,}bin/sed mrix,
+  /{usr/,}lib/dhcpcd/dhcpcd-run-hooks mrix,
```

Comment:
Use `rix` instead or `ix` or `mrix`

Reply:
Also as they are a lot of possible shell, you should use   `/{usr/,}bin/{,ba,da}sh  rix,`, instead of `/{usr/,}bin/bash`